chef_master/source/knife_cookbook_site.rst

```diff
@@ -17,7 +17,7 @@ The ``knife cookbook site`` subcommand is used to interact with cookbooks that a
 
 .. warning:: .. tag notes_knife_cookbook_site_use_devkit_berkshelf
 
-             Please consider managing community cookbooks using the version of Berkshelf that ships with the Chef development kit. For more information about the Chef development kit, see `About the Chef DK </about_chefdk.html>`__.
+             Please consider managing community cookbooks using the version of Berkshelf that ships with the Chef Development Kit. For more information about the Chef Development Kit, see `About the Chef DK </about_chefdk.html>`__.
```

Comment:
We've discussed taking a stronger editorial position on using Chef, which means using more direct language. For example:  "Use Berkshelf, which is part of the Chef Development Kit, to manage community cookbooks. For more information about the Chef Development Kit, see `About the Chef DK </about_chefdk.html>`__."
or "Chef recommends using Berkshelf..."

Reply:
I'm fine with moving beyond a friendly suggestion. In reality we need to push people to policyfiles, but I think that's a project and not something we can do here.